package.json

```
@@ -17,7 +17,7 @@
         "watch": "jbuild watch"
     },
     "dependencies": {
-        "js-yaml": "4.0.x",
+        "js-yaml": "4.1.x",
         "ports": "1.1.x",
         "underscore": "1.12.x"
```

Would be great to have more flexibility:

```suggestion
        "js-yaml": "^4.1.1",
        "ports": "^1.1.0",
        "underscore": "^1.13.7"
```

`ports` hasn't been updated in 12 years, `underscore@1.12.1` is 5 years old

At least for `underscore` something can be done, `1.13.7` is 16 months old, in 2 months it will also start triggering security warnings, but at least it will be easier to justify them.